### PR TITLE
NAACL 2019: Added corrected video links to anthology.

### LINKED ISSUE #763: NAACL 2019 videos
NAACL 2019 videos [are uploaded](https://vimeo.com/manage/folders/820515), but unfortunately not according [to spec](https://github.com/acl-org/acl-anthology/wiki/Contract-Guidelines-for-Recording-Talks). @akoehn, you had code for matching on titles, right? Would you mind trying to do the linkup sometime if you get a free hour?

### PR DESCRIPTION
**Pull request by SpencerWhitehead**
Added in the corrected NAACL 2019 video links to the anthology XML.
Fixes #763

### CODE REVIEW

File: data/xml/N19.xml [hunk 1]
```diff
@@ -4,9 +4,18 @@
...
+      </editor>
+      <editor>
+        <first>Christy</first>
+        <last>Doran</last>
+      </editor>
+      <editor>
+        <first>Thamar</first>
+        <last>Solorio</last>
+      </editor>
       <publisher>Association for Computational Linguistics</publisher>
```
Comment (mjpost) [inline data/xml/N19.xml]: I'm not sure how you created this, but it broke our formatting, which puts `<author>` and `<editor>` tags one per line. See the [indent function](https://github.com/acl-org/acl-anthology/blob/master/bin/anthology/utils.py) which should fix this for you.

**Comment by mjpost** [comment]
I forgot that there is also a script, [indent.py](https://github.com/acl-org/acl-anthology/blob/master/bin/indent.py), that you could call on the XML file from the command line.

File: data/xml/N19.xml (hunk 1 above)
Comment (mjpost) [reply data/xml/N19.xml]: Thanks for the push! Still not quite right. `git diff` should only show changes on the `<video>` lines....

File: data/xml/N19.xml [hunk 2]
```diff
@@ -2915,7 +3021,7 @@
     <paper id="264">
       <title>Guiding Extractive Summarization with Question-Answering Rewards</title>
       <author><first>Kristjan</first><last>Arumae</last></author>
-      <author id="fei-liu-utdallas"><first>Fei</first><last>Liu</last></author>
+      <author><first>Fei</first><last>Liu</last></author>
       <pages>2566–2577</pages>
```
Comment (mjpost) [inline data/xml/N19.xml]: Do you know how this happened? I wonder if you should start over wit this PR.

File: data/xml/N19.xml [hunk 3]
```diff
@@ -3346,17 +3465,20 @@
...
       <url>N19-1302</url>
       <doi>10.18653/v1/N19-1302</doi>
+      <video href="https://vimeo.com/356097901" tag="video"/>
     </paper>
     <paper id="303">
       <title><fixed-case>G</fixed-case>ender<fixed-case>Q</fixed-case>uant: Quantifying Mention-Level Genderedness</title>
-      <author><first/><last>Ananya</last></author>
+      <author><first/>
+<last>Ananya</last></author>
       <author><first>Nitya</first><last>Parthasarthi</last></author>
```
Comment (mjpost) [inline data/xml/N19.xml]: Remaining error.

File: data/xml/N19.xml [hunk 4]
```diff
@@ -4820,7 +5010,7 @@
       <abstract>There have been many studies on neural headline generation models trained with a lot of (article, headline) pairs. However, there are few situations for putting such models into practical use in the real world since news articles typically already have corresponding headlines. In this paper, we describe a practical use case of neural headline generation in a news aggregator, where dozens of professional editors constantly select important news articles and manually create their headlines, which are much shorter than the original headlines. Specifically, we show how to deploy our model to an editing support tool and report the results of comparing the behavior of the editors before and after the release.</abstract>
       <url>N19-2010</url>
       <doi>10.18653/v1/N19-2010</doi>
-      <attachment type="presentation">N19-2010.Presentation.pdf</attachment>
+      <video href="https://vimeo.com/361705360" tag="video"/>
     </paper>
```
Comment (mjpost) [inline data/xml/N19.xml]: Somehow this attachment was removed.

File: data/xml/N19.xml [hunk 5]
```diff
@@ -5264,7 +5459,7 @@
     </frontmatter>
     <paper id="1">
       <title>Abbreviation Explorer - an interactive system for pre-evaluation of Unsupervised Abbreviation Disambiguation</title>
-      <author><first>Manuel R.</first><last>Ciosici</last></author>
+      <author><first>Manuel</first><last>Ciosici</last></author>
       <author><first>Ira</first><last>Assent</last></author>
```
Comment (mjpost) [inline data/xml/N19.xml]: Name mangled.

File: data/xml/N19.xml [hunk 6]
```diff
@@ -5210,7 +5404,6 @@
       <abstract>In this paper we study how different ways of combining character and word-level representations affect the quality of both final word and sentence representations. We provide strong empirical evidence that modeling characters improves the learned representations at the word and sentence levels, and that doing so is particularly useful when representing less frequent words. We further show that a feature-wise sigmoid gating mechanism is a robust method for creating representations that encode semantic similarity, as it performed reasonably well in several word similarity datasets. Finally, our findings suggest that properly capturing semantic similarity at the word level does not consistently yield improved performance in downstream sentence-level tasks.</abstract>
       <url>N19-3016</url>
       <doi>10.18653/v1/N19-3016</doi>
-      <attachment type="poster">N19-3016.Poster.pdf</attachment>
     </paper>
```
Comment (mjpost) [inline data/xml/N19.xml]: Attachment removed.

**Comment by mjpost** [comment]
There are a number of remaining stray requests. You may want to revert to master, push that up, then reapply your changes after calling `indent` (which shouldn't have any of the odd deletions).

File: data/xml/N19.xml (hunk 4 above)
Comment (SpencerWhitehead) [reply data/xml/N19.xml]: Thanks for the heads up! I just noticed that and corrected it in the latest version. I have one more change to make and then it should be done. I'm not sure what happened with my update script, but it dropped a few things for some odd reason. I'll have this resolved momentarily.

File: data/xml/N19.xml [hunk 7]
```diff
@@ -209,15 +220,17 @@
...
       <url>N19-1018</url>
       <doi>10.18653/v1/N19-1018</doi>
+      <video href="https://vimeo.com/360494509" tag="video"/>
     </paper>
     <paper id="19">
-      <title><fixed-case>H</fixed-case>ow <fixed-case>B</fixed-case>ad are <fixed-case>P</fixed-case>o<fixed-case>S</fixed-case> <fixed-case>T</fixed-case>agger in <fixed-case>C</fixed-case>ross-<fixed-case>C</fixed-case>orpora <fixed-case>S</fixed-case>ettings? <fixed-case>E</fixed-case>valuating <fixed-case>A</fixed-case>nnotation <fixed-case>D</fixed-case>ivergence in the <fixed-case>UD</fixed-case> <fixed-case>P</fixed-case>roject.</title>
+      <title><fixed-case>H</fixed-case>ow <fixed-case>B</fixed-case>ad are <fixed-case>P</fixed-case>o<fixed-case>S</fixed-case><fixed-case>T</fixed-case>agger in <fixed-case>C</fixed-case>ross-<fixed-case>C</fixed-case>orpora <fixed-case>S</fixed-case>ettings? <fixed-case>E</fixed-case>valuating <fixed-case>A</fixed-case>nnotation <fixed-case>D</fixed-case>ivergence in the <fixed-case>UD</fixed-case><fixed-case>P</fixed-case>roject.</title>
       <author><first>Guillaume</first><last>Wisniewski</last></author>
```
Comment (mjpost) [inline data/xml/N19.xml]: Looks like there is one more inadvertent change.

File: data/xml/N19.xml (hunk 1 above)
Comment (SpencerWhitehead) [reply data/xml/N19.xml]: You're absolutely correct and thanks for pointing that out! The issues should be fixed and the only differences should be the video lines.

For the future, I'll see what I can do to resolve that issue in my code that does the adding of the videos to the XML to make sure the formatting is correct.

Comment (mjpost) [reply data/xml/N19.xml]: Can you add your script under `bin/` so others can use it in the future? How does it relate to Arne's implementation?

**Comment by SpencerWhitehead** [comment]
@mjpost I apologize for this flurry of commits, but I believe I have resolved the errors. However, to keep things clean, would you still like me to delete this PR, revert to master, then reapply the changes?

**Comment by mjpost** [comment]
If it's correct, it's fine—we squash when merging, anyway, so it will be just a single commit.

**Comment by SpencerWhitehead** [comment]
Okay, yeah it should be correct. After this, I'll try to fix my code so it directly outputs the correct format. Thanks!

**Comment by mjpost** [comment]
closes #763

**Pull request merged**